content/articles/verifying-domain-resolution.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Verifying Domain Resolution
+excerpt: This article explains the different ways you can verify that a domain is resolving.
+categories:
+- Domains
+---
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+# Verifying Domain Resolution
+
+Certain changes to your domain require that its resolution be properly configured. To ensure this, we run checks to verify that your domain is resolving correctly before you make any changes. If we detect anything that might prevent a change from taking effect, a notification will appear listing the failed verifications and the actions you need to take to get your domain resolving.
+
+![screenshot: the domain resolution notificatoin](/files/verifying-domain-resolution-notification.png)
+
+## Domain is not registered
+
+Your domain needs to be registered for your changes to take effect. You can register the domain at DNSimple by clicking **Register domain** in the notification. Refer to our [guide](/articles/registering-domain/) on domain registration for more information.
+
+## The DNS zone is inactive
+
+Your domain's DNS zone needs to be active for the domain to resolve and for your changes to take effect. You can activate the DNS zone for your domain by clicking **Activate zone** in the notification. Refer to our [guide](/articles/dns-hosting/#activating-a-dns-zone) on activating a DNS zone for more information.
```

Comment:
Is it worth saying something even more general about how you get in this mess – like:

```suggestion
When a customer isn't using their zone, they may decide to deactivate the zone to avoid being billed it. When you resume using it, re-activating it can easily be overlooked.

Your domain's DNS zone needs to be active for the domain to resolve and for your changes to take effect. You can activate the DNS zone for your domain by clicking **Activate zone** in the notification. Refer to our [guide](/articles/dns-hosting/#activating-a-dns-zone) on activating a DNS zone for more information.
```

Reply:
Edits to make this line up with the rest of the article (you vs they): 

When you're not using a zone, you may decide to deactivate it to avoid being billed. When you resume using it, re-activating it can easily be overlooked.

Reply:
```suggestion
When you're not using a zone, you may decide to deactivate it to avoid being billed. When you resume using it, re-activating it can easily be overlooked.

Your domain's DNS zone needs to be active for the domain to resolve and for your changes to take effect. You can activate the DNS zone for your domain by clicking **Activate zone** in the notification. Refer to our [guide](/articles/dns-hosting/#activating-a-dns-zone) on activating a DNS zone for more information.
```